uconnrcmpy/conditions.py

```diff
@@ -852,9 +852,14 @@ def compare_to_sim(self, run_reactive=False, run_nonreactive=True):
             copy_str += '{}'.format(ignition_delay)
 
         if self.nonreactive_sim is not None and self.reactive_sim is not None:
-            # Use np.where to find the index of EOC for reqctive simulation
-            # Use that index to find T_EOC_reactive
+            # Use np.where to find the index of EOC for reactive simulation
+            # Use the given index to find T_EOC_reactive
             # Compare to T_EOC_nonreactive using np.isclose
+            EOC_reactive_idx = np.where(self.reactive_sim.time >= compression_time)
+            T_EOC_reactive = self.reactive_sim.temperature[EOC_reactive_idx][0]
```

Comment:
I think I made a typo here, it should be `self.reactive_sim.temperature[EOC_reactive_idx[0]]`... although now I think about it, it doesn't matter, actually...